src/pages/DatasetStatistics.tsx

```diff
@@ -111,11 +128,11 @@ export default function DatasetStatistics(props) {
           </div>
           <div style={Styles.SUB_HEADER}>Dataset Information</div>
           <div style={{display: 'flex'}}>
-            <div style={Styles.DESCRIPTION_BOX}>
+            <div style={Styles.DESCRIPTION_BOX as React.CSSProperties}>
               <div style={{...Styles.MINOR_HEADER, paddingLeft: '10px'}}>Dataset Description:</div>
               {LINE}
               <div style={{fontSize: Theme.font.size.small, padding: '1rem'}}>
-                {extract('Dataset Description') || dataset?.description || dataset?.study?.description || 'N/A'}
+                {extract('Dataset Description') || dataset?.study?.description || 'N/A'}
```

Comment:
Why is `dataset?.description` not needed here?

Reply:
I think that was an error, the type `Dataset` doesn't have a `description` field. Maybe the API changed at some point and this was kept around for backward compatibility?

Reply:
Its not actually a field on the typed object! I don't know why it was there before.